components/http/okHttp/src/main/java/com/microsoft/kiota/http/ContinuousAccessEvaluationClaims.java

```diff
@@ -0,0 +1,56 @@
+package com.microsoft.kiota.http;
+
+import jakarta.annotation.Nonnull;
+import jakarta.annotation.Nullable;
+
+import okhttp3.Response;
+
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Helper class to extract the claims from the WWW-Authenticate header in a response.
+ * https://learn.microsoft.com/en-us/entra/identity/conditional-access/concept-continuous-access-evaluation
+ */
+public final class ContinuousAccessEvaluationClaims {
```

Comment:
do we need this class to be public? can't we restrain it to package so we're not shipping additional API on this side?

Reply:
It's being used in the `com.microsoft.kiota.http` package and `com.microsoft.kiota.http.middleware` package. 
Restricting it would make it only visible in the main `http` package and not in the `middleware` package and vice versa. Java doesn't have an equivalent of `internal` in .NET

Reply:
Thanks for clarifying here.